client/lib/parsing.rs

```diff
@@ -339,13 +339,23 @@ pub(super) fn parse_deploy_params(
     gas_price: &str,
     dependencies: &[&str],
     chain_name: &str,
+    session_account: &str,
 ) -> Result<DeployParams> {
     let secret_key = self::secret_key(secret_key)?;
     let timestamp = self::timestamp(timestamp)?;
     let ttl = self::ttl(ttl)?;
     let gas_price = self::gas_price(gas_price)?;
     let dependencies = self::dependencies(dependencies)?;
     let chain_name = chain_name.to_string();
+    let session_account = if !session_account.is_empty() {
+        let public_key = PublicKey::from_hex(session_account).map_err(|error| {
+            eprintln!("Can't parse the contents as a public key: {}", error);
```

Comment:
Lets not print to stderr in the lib please.  That's OK in the binary (for now), but not the lib.

Reply:
Removed the print to stderr